### PR TITLE
added an additional task to the lesson on using visualisation with ggplot2

### PR DESCRIPTION
**Pull request by skyfallen**
I have added few questions and examples to the ggplot2 lesson in R-ecology. Looking for your reply.

### CODE REVIEW

File: 05-visualisation-ggplot2.Rmd [hunk 1]
```diff
@@ -285,4 +285,36 @@ ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group
...
+  facet_wrap(~ species_id) + 
+  labs(title = 'Observed species in time', 
+       x = 'Year of observation', 
+       y = 'Number of species') + 
+  theme(text=element_text(size=16, family="Arial"))
+```
+
+Now, labels became bigger, but there are still few things that one could be improved. Please, take another five minutes and try to add another one or two things, to make it look even more beautiful. Use ggplot2 cheat sheet, which we linked earlier for inspiration.
+
+Here are some ideas from me:
```
Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: please change to: "Here are some ideas:"

File: 05-visualisation-ggplot2.Rmd [hunk 2]
```diff
@@ -285,4 +285,36 @@ ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group
...
+
+* See if you can change thickness of the lines.
+* Can you find a way to change the name of the legend? What about its labels?
+* Using different colour palette improve the look? (http://www.cookbook-r.com/Graphs/Colors_(ggplot2)/)
```
Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: change to: Use different color palette to improve the look (http://www.cookbook-r.com/Graphs/Colors_(ggplot2)/)

File: 05-visualisation-ggplot2.Rmd [hunk 3]
```diff
@@ -285,4 +285,36 @@ ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group
...
+* Advanced: Can you turn x-axis labels by some degree so that they won't overlap? Try 90 degrees angel and then experiment more to get nicely looking diagonally placed labels.
```
Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: replace "Advanced" by "Bonus"

Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: 90 degree angle

Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: Rephrase: Let's change the orientation of the labels so they don't overlap. You can use a 90 degree angle, or experiment to find the appropriate angle for diagonally oriented labels.

File: 05-visualisation-ggplot2.Rmd [hunk 4]
```diff
@@ -285,4 +285,36 @@ ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group
 ```{r average-weight-time-facet_sex}
 ggplot(data = yearly_weight, aes(x=year, y=avg_weight, color = species_id, group = species_id)) +
   geom_line() + facet_wrap(~ sex)
-```
\ No newline at end of file
+```
+
+So far our result looks quite good, but it is yet far from being publishable. What are other ways one can improve it? Take a look at the ggplot2 cheat sheet (https://www.rstudio.com/wp-content/uploads/2015/08/ggplot2-cheatsheet.pdf), and write down at least three more ideas (can leave them as comments to Etherpad.
```
Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: * missing closing parenthesis after Etherpad.
* (**you** can leave them as comments in the Etherpad)

**Comment by fmichonneau** [comment]
Thanks for your PR @skyfallen!

Can you please: 
* submit only the Rmd file (the png and the html files will be generated later)?
* wrap lines to 80 characters. If you text doesn't let you do this easily, you can use the function `wrap_rmd` from the knitr package.
* make sure there is a blank line before and after each chunk (otherwise the text gets placed next to the plot)
* address the inline suggestions.

**Comment by skyfallen** [comment]
Great, thanks for the thorough review will submit fixed version asap!

**Comment by skyfallen** [comment]
I added only the edited Rmd file. How can I make sure that only Rmd file gets submitted?

File: 05-visualisation-ggplot2.Rmd [hunk 5]
```diff
@@ -26,11 +26,12 @@ capabilities in basic R, but ggplot2 adds more powerful plotting capabilities.
...
 >
 > -	Visualize some of the
->[mammals data](http://figshare.com/articles/Portal_Project_Teaching_Database/1314459)
+>[mammals
+data](http://figshare.com/articles/Portal_Project_Teaching_Database/1314459)
 >from Figshare [surveys.csv](http://files.figshare.com/1919744/surveys.csv)
 > -	Understand how to plot these data using R ggplot2 package. For more details
->on using ggplot2 see
->[official documentation](http://docs.ggplot2.org/current/).
+>on using ggplot2 see >[official
```
Comment (fmichonneau) [inline 05-visualisation-ggplot2.Rmd]: It looks like there is a > in the middle of the line

**Comment by fmichonneau** [comment]
Hi @skyfallen, to not include the html and png files, you can rebase your branch to only commit the Rmd file, or create a new PR only with the Rmd file. Once you address the little typo I mentioned, it looks like your changes will be ready to be merged.

**Comment by fmichonneau** [comment]
closed in favor of #57

**Pull request closed without merging**